File: eRent.Models/FitPasosModel.cs
using System;

namespace eRent.Models
{
    public class FitPasosModel
    {
        public int PasosId { get; set; }

        public int KorisnikId { get; set; }

        public DateTime DatumIzdavanja { get; set; }

        public bool IsValid { get; set; }
        public KorisnikModel Korisnik { get; set; }
    }
}

File: mobile/myapp/lib/model/FitPasosModel.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:myapp/model/korisnik.dart';

part 'FitPasosModel.g.dart';

@JsonSerializable()
class FitPasos {
  int? pasosId;
  int? korisnikId;
  DateTime? datumIzdavanja;
  DateTime? datumIsteka;
  bool? isValid;
  Korisnik? korisnik;

  FitPasos();

  factory FitPasos.fromJson(Map<String, dynamic> json) =>
      _$FitPasosFromJson(json);

  /// Connect the generated [_$ProductToJson] function to the `toJson` method.

  Map<String, dynamic> toJson() => _$FitPasosToJson(this);
}
